hera_mc/cm_handling.py

```diff
@@ -153,7 +153,7 @@ def __find_ports(self, connections_dict, current):
                 elif ':down:' in k and v.upstream_output_port not in output_ports:
                     output_ports.append(v.upstream_output_port)
                 else:
-                    print("cm_handling[146]: ERROR SHOULD BE UP or DOWN ", k)
+                    print("cm_handling[156]: WARNING SHOULD BE UP or DOWN ", k)
```

Comment:
I changed it to a Warning from an Error, because it reflects something fishy in the database, but the ending result is (generally) fine.  I updated the text in the warning a bit.  (I've already spent too much time trying to debug that one, since it is a funny interaction of things...)